ibm/mas_devops/roles/mongodb/tasks/main.yml

```diff
@@ -12,6 +12,7 @@
   when:
     - mongodb_provider == "community"
     - mongodb_action != "none"
+    - mongodb_action != "update"
```

Comment:
There is no such action as update for mongo, we just call install again don't we?

Reply:
changed "update" to "upgrade" which is consistent with other roles and created an "upgrade" action.